src/const.ts

```diff
@@ -65,6 +65,7 @@ export const DEFAULT_CONSTANT = {
   enableEmojiFeedback: true,
   enableMention: true,
   enableResetHistoryOnConnect: false,
+  enableHideWidgetForDeactivatedUser: false,
```

Comment:
if user is deactivated and enableHideWidgetForDeactivatedUser is false, what happens when widget is open? (can channel be fetched? is message input disabled? etc)

Reply:
connection fails and nothing will work
<img width="423" alt="image" src="https://github.com/sendbird/chat-ai-widget/assets/26326015/5640ac0f-ec05-4fff-a38f-49bb23248f84">

Reply:
hmm not sure when client would want this. Hence maybe enableHideWidgetForDeactivatedUser is not necessary. What do you think?

Reply:
This feature determines whether to render the button after the connection is made, resulting in a slight delay equal to the connection time in rendering the button. Therefore, it should not be enabled by default.

Reply:
ok but I think just displaying "somethign went wrong retry button" in case of deactivated user is misleading. We have to do something about this.

Reply:
Usually, it is rare to deactivate specific users while using the widget.
If there is a request to hide the widget from malicious users, instructions on how to use this feature and deactivate users will be provided.